.github/workflows/release.yml

```diff
@@ -110,17 +110,17 @@ jobs:
           ref: 'main'
           fetch-depth: 0
 
-      - name: Setup Python
-        uses: actions/setup-python@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
         with:
-          python-version: '3.8'
+          node-version: 12
```

Comment:
we should use latest / 16 unless we have a reason for 12.

Reply:
yes please